src/main/java/com/kryptnostic/datastore/cassandra/CommonColumns.java

```diff
@@ -30,19 +30,22 @@
 
 public enum CommonColumns implements ColumnDef {
     ACLID( DataType.uuid() ),
+    AUDIT_ID( DataType.uuid() ),
     FROZEN_PERMISSIONS( DataType.frozenSet( DataType.text() ) ),
     ACL_CHILDREN_PERMISSIONS( DataType.map( DataType.uuid(), FROZEN_PERMISSIONS.getType() ) ),
+    // TODO: ACL_KEYS should really be ACL_KEY
     ACL_KEYS( DataType.frozenList( DataType.uuid() ) ), // partition index within a table for distribution purpose,
     ACL_ROOT( DataType.frozenList( DataType.uuid() ) ),
     ALLOWED_EMAIL_DOMAINS( DataType.set( DataType.text() ) ),
     AUDIT_EVENT_DETAILS( DataType.text() ),
-    AUDIT_EVENT_TYPE( DataType.text() ),
     BIDIRECTIONAL( DataType.cboolean() ),
+    BLOCK_ID( DataType.uuid() ),
     CLOCK( DataType.timestamp() ),
     DATATYPE( DataType.text() ),
     DESCRIPTION( DataType.text() ),
     DEST( DataType.set( DataType.uuid() ) ),
     ENTITYID( DataType.text() ),
+    ENTITY_ID( DataType.uuid() ),
```

Comment:
@geekbeast you said entity ID is just a vertex ID for the corresponding `AclKey`. Should `ENTITY_ID` be a `UUID`?